openapi/definitions.yaml

```diff
@@ -161,6 +161,9 @@ LegalData:
     original_message_url:
       type: string
       minLength: 1
+    pec_server_service_id:
```

Comment:
is it optional?

Reply:
Yes, it will be auto-filled by io-fn-services at message creation.
For the legal message already stored, it will fallback using the Poste pec server as default.